tests/foreman/api/test_contentmanagement.py

```diff
@@ -76,6 +77,28 @@ def test_positive_sync_repos_with_large_errata(self):
             with self.assertNotRaises(TaskFailedError):
                 repo.sync()
 
+    @tier2
+    def test_positive_sync_repos_with_lots_files(self):
+        """Attempt to synchronize repository containing a lot of files inside
```

Comment:
but that repo (https://abalakht.fedorapeople.org/test_repos/lots_files/) contain only one file

Reply:
It contains 1 package, that's true, as it's better to isolate the issue rather than sync yet another huge repo like rhel7. Anyway, package != file. Although this repo contains 1 package, there's ~ 66920 files in it.